cookbooks/clearwater/recipes/local_config.rb

```diff
@@ -51,15 +51,18 @@
   # Set up an array of all the sites.
   sites = Array.new(number_of_sites)
   for i in 0...number_of_sites
-      sites[i] = "site#{i}"
+      sites[i] = "site#{i+1}"
   end
 
   # Work out which site this node is in based on its index.
   local_site_index = node[:clearwater][:index] % number_of_sites
-  local_site = sites[local_site_index]
+  if local_site_index == 0
+    local_site_index = number_of_sites
+  end
```

Comment:
It would be nice if we could compress these 4 lines into 1 line, especially given that we've had to write them a few times, but I can't think of a sensible way to do it. It could be worth turning it into a function but we need to use it in two different files and there's currently no obvious place to put this potential function (and I'm not sure it actually adds a great deal of value - it would just make the code slightly less bloated). Meh...